.github/workflows/ci-slang-build.yml

```diff
@@ -170,35 +170,39 @@ jobs:
           # Copy artifacts from the specific configuration directory
           if [ -d "build/${cmake_config}" ]; then
             echo "Copying artifacts from build/${cmake_config} to github_artifact/${cmake_config}..."
-            
+
             # Copy bin directory excluding debug files
             if [ -d "build/${cmake_config}/bin" ]; then
               mkdir -p "artifacts-staging/${cmake_config}/bin"
               find "build/${cmake_config}/bin" -type f ! -name "*.pdb" ! -name "*.ilk" ! -name "*.dwarf" ! -name "*.debug" ! -name "*.dSYM" -exec cp {} "artifacts-staging/${cmake_config}/bin/" \; || echo "Failed to copy bin directory"
 
               cp -r "build/${cmake_config}/bin/D3D12" "artifacts-staging/${cmake_config}/bin/" || echo "Failed to copy bin/D3D12 directory"
+              if [[ "${{ inputs.os }}" == "windows" ]]; then
```

Comment:
We should test the release pipeline and ensure the produced package can work as expected. You can do so by pushing your branch to a branch in the official repo, and create a test release from that branch.